src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -451,6 +451,50 @@ private function isSimpleInstance($id, Definition $definition)
     }
 
     /**
+     * Checks if the definition is a trivial instance.
+     *
+     * @param Definition $definition
+     *
+     * @return bool
+     */
+    private function isTrivialInstance(Definition $definition)
```

Comment:
Well, if in the future we decide that more cases would benefit from this, we could still change the defining of what a trivial service is. And we can decide to rename the method if actually necessary.

`canBeInlined` has another reason to be confusing: we have another concept called `inlining` in the component (although a bit related)